qiskit/pulse/__init__.py

```diff
@@ -145,8 +137,254 @@
+    breaking API changes without deprecation warnings in future releases until
+    otherwise indicated.
+
+The pulse builder provides an imperative API for writing pulse programs
+with less difficulty than the :class:`~qiskit.pulse.Schedule` API.
+It contextually constructs a pulse schedule and then emits the schedule for
+execution. For example to play a series of pulses on channels is as simple as:
+
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.visualization import SchedStyle
+
+    dc = pulse.DriveChannel
+    d0, d1, d2, d3, d4 = dc(0), dc(1), dc(2), dc(3), dc(4)
+
+    with pulse.build(name='pulse_programming_in') as pulse_prog:
+        pulse.play([1, 1, 1, 0, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 0, 1, 0, 1, 1, 1, 0, 1, 1, 1], d0)
+        pulse.play([1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0], d1)
+        pulse.play([1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0], d2)
+        pulse.play([1, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0], d3)
+        pulse.play([1, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 0, 1, 0, 1, 1, 1, 0, 0, 1, 0], d4)
+
+    style = SchedStyle(figsize=(5, 2.5))
+    pulse_prog.draw(style=style)
+
+
+In the future the pulse builder will be coupled to the
+:class:`~qiskit.circuit.QuantumCircuit` with an equivalent circuit builder
+interface.
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ~qiskit.pulse.builder.build
+
+
+Channels
+--------
+Methods to return the correct channels for the respective qubit indices.
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.test.mock import FakeArmonk
+
+    backend = FakeArmonk()
+
+    with pulse.build(backend) as drive_sched:
+        d0 = pulse.drive_channel(0)
+        print(d0)
+
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ~qiskit.pulse.builder.acquire_channel
+    ~qiskit.pulse.builder.control_channels
+    ~qiskit.pulse.builder.drive_channel
+    ~qiskit.pulse.builder.measure_channel
+
+
+Instructions
+------------
+Pulse instructions are available within the builder interface. Here's an example:
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.test.mock import FakeArmonk
+    from qiskit.visualization import SchedStyle
+
+    backend = FakeArmonk()
+
+    with pulse.build(backend) as drive_sched:
+        d0 = pulse.drive_channel(0)
+        a0 = pulse.acquire_channel(0)
+
+        pulse.play(pulse.library.Constant(10, 1.0), d0)
+        pulse.delay(20, d0)
+        pulse.shift_phase(3.14/2, d0)
+        pulse.set_phase(3.14, d0)
+        pulse.shift_frequency(1e7, d0)
+        pulse.set_frequency(5e9, d0)
+
+        with pulse.build() as temp_sched:
+            pulse.play(pulse.library.Gaussian(20, 1.0, 3.0), d0)
+            pulse.play(pulse.library.Gaussian(20, -1.0, 3.0), d0)
+
+        pulse.call(temp_sched)
+        pulse.acquire(30, a0, pulse.MemorySlot(0))
+
+    style = SchedStyle(figsize=(5, 2.5))
```

Comment:
```suggestion
    style = SchedStyle(figsize=(3, 2), title_font_size=10, axis_font_size=8, icon_font_size=10)
```
![image](https://user-images.githubusercontent.com/11485594/87706506-a44b9180-c76d-11ea-8ffe-3dc2289710c0.png)

Reply:
Fixed in 0194a2960.